crates/rattler_package_streaming/src/fs.rs

```diff
@@ -0,0 +1,51 @@
+use crate::{ArchiveType, ExtractError};
+use std::fs::File;
+use std::path::Path;
+
+/// Extracts the contents a `.tar.bz2` package archive from the specified remote location.
```

Comment:
remote location? I interpret the function as taking a `fs::path`?

Reply:
Changed